packages/shared/src/components/Feed.tsx

```diff
@@ -207,7 +213,7 @@ export default function Feed<T>({
 
   const useList = insaneMode && numCards > 1;
   const virtualizedNumCards = useList ? 1 : numCards;
-
+  const bookmarkLoops = useFeature(feature.bookmarkLoops);
```

Comment:
This will enroll users that dont bookmark, can you check with Nensi/ido what the rule is?

Reply:
Right. Very good point. Let me check.

Reply:
As Ido mentioned in thread, only users who bookmark, for this Lee u can use this hook @rebelchris created `useConditionalFeature`, u might need a way to know though that keeps track if a user bookmarked after this is merged, maybe actions?